### PR TITLE
Fix support big integers. Closes #13

### LINKED ISSUE #13: Support big integers
Redis uses 64 bit signed ints -> the max value is 9,223,372,036,854,775,807. Since we store scheduled rewards as attoFIL, the maximum value in FIL would be ~9.22 FIL (per user). This obviously won't work.

Rearchitect to store the values as strings:
- either use transactions to atomically read+incr+write the value (using BigInt())
- or store one value per increase and then sum when reading

### PR DESCRIPTION
**Pull request by juliangruber**
Closes #13

### CODE REVIEW

**Comment by juliangruber** [comment]
We need to support concurrent writes as it's possible for `spark-evaluate` to increase scheduled rewards at the same time as `POST /paid` (called from CLI) will decrease them

**Comment by juliangruber** [comment]
@bajtos hindsight please

File: index.js [hunk 1]
```diff
@@ -112,38 +113,59 @@ async function handleIncreaseScores (req, res, redis, signerAddresses) {
...
-    addLogJSON(tx, {
-      timestamp,
-      address,
-      score,
-      scheduledRewardsDelta: String(scheduledRewards)
+  if (body.participants.length === 0) {
+    return json(res, {})
+  }
+
+  const scheduledRewardsDelta = body.scores.map(score => {
```
Comment (bajtos) [inline index.js:120]: I find it a bit surprising that a variable with singular name ("...Delta") is storing an array of values.

`scheduledRewardsDeltas` would be a better name (IMO).

**Review by bajtos — COMMENTED** [review]
I don't have enough bandwidth to review this change in depth. I don't see any obvious issues.

File: index.js (hunk 1 above)
Comment (juliangruber) [reply index.js:120]: +1 https://github.com/filecoin-station/spark-rewards/commit/db9a5aa586a3cfa4a8104f6dcabddc06ca95a88b

**Pull request merged**